src/main/java/com/mraof/minestuck/alchemy/CardCaptchas.java

```diff
@@ -117,17 +152,12 @@ private String createCaptchaForItem(Item item, long seed)
 		String cutHash = shuffledHash.substring(shuffledHash.length() - 16); //last 16 characters of hash
 		String captcha = captchaFromHash(cutHash);
 		
-		if(captchasMap.containsValue(captcha))
+		if(captchasMap.containsValue(captcha) || PredeterminedCardCaptchas.GetData().containsValue(captcha))
```

Comment:
I see that generated captchas are also checked in `generateBackupCaptcha()`. Should we check predetermined captchas there too?